lib/realtime/vehicle.ex

```diff
@@ -119,8 +138,9 @@ defmodule Realtime.Vehicle do
       timestamp: VehiclePosition.last_updated(vehicle_position),
       latitude: VehiclePosition.latitude(vehicle_position),
       longitude: VehiclePosition.longitude(vehicle_position),
-      direction_id: TripUpdate.direction_id(trip_update),
-      route_id: TripUpdate.route_id(trip_update),
+      direction_id:
+        VehiclePosition.direction_id(vehicle_position) || TripUpdate.direction_id(trip_update),
+      route_id: VehiclePosition.route_id(vehicle_position) || TripUpdate.route_id(trip_update),
```

Comment:
Same question as above: Does Swiftly get its `route_id` and `direction_id` from the trip, and would we ever prefer Swiftly's vs GTFS's?

If we always use GTFS's, we don't need to parse this field from the swiftly response.